website/docs/r/fsx_file_system.html.markdown

```diff
@@ -0,0 +1,91 @@
+---
+layout: "aws"
+page_title: "AWS: aws_fsx_file_system"
+sidebar_current: "docs-aws-resource-fsx-file-system"
+description: |-
+  Provides an FSx filesystem resource.
+---
+
+# aws_fsx_file_system
```

Comment:
This was likely done after this pull request was originally submitted but we are prefixing the titles with `Resource: ` and `Data Source: ` currently.

```suggestion
# Resource: aws_fsx_file_system
```